api/v1/routes/payment.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import APIRouter
+from api.v1.services.payment import PaymentService
+from api.db.database import get_db
+from sqlalchemy.orm import Session
+from fastapi import Depends, status
+# from api.utils.dependencies import get_current_user
+from api.v1.models.user import User
+from api.v1.services.user import user_service
+from api.v1.schemas.payment import PaymentSchema, CreatePaymentSchema
+from api.v1.schemas.token import TokenData
+from typing import List
+
+payment = APIRouter(prefix='/payments', tags=['Payment'])
+
+
+# @payment.get('/{payment_id}', status_code=status.HTTP_200_OK)
+# async def get_single_payment(payment_id: str, db: Session = Depends(get_db), current_user: User = Depends(user_service.get_current_user)):
+#     payment = PaymentService.fetch(db, payment_id)
+#     return payment
```

Comment:
you commented this out

Reply:
Yes I'm creating two endpoints for separate tickets, I'm pushing the commented sections in a separate PR

Reply:
@Goketech has this been resolved, please don't comment out code. Anything that is not been used should be removed

Reply:
Yes it has been resolved already @bolexs